app/Config/Modules.php

```diff
@@ -72,7 +72,7 @@ class Modules extends BaseModules
      *
      * If it is not listed, only the base application elements will be used.
      *
-     * @var string[]
+     * @var list<string>
```

Comment:
If `list` is okay with `@var`, we can remove `@phpstan-var` tags like:
```
     * @var array<int, string>
     * @phpstan-var list<string>
```

Reply:
Interesting. I think vscode intelephense has now support for this type, as well as Closure signatures.

Reply:
Okay, then let's remove `@phpstan-var` from now on.

Reply:
I'm all for this; I can confirm that Intelephense (in BBEdit) supports `list` as well.

Reply:
I know I always ask this... these are different, right?
```
list<string>
string[]
```

I think the latter just means "values are all strings" without any indication of keys. But I can't find the answer explicitly. 

* https://phpstan.org/writing-php-code/phpdoc-types#general-arrays
* https://docs.phpdoc.org/2.9/guides/types.html#arrays

Reply:
Yes.
> `/** @return ValueType[] */`
> In Psalm this annotation is equivalent to @psalm-return array<array-key, ValueType>.
https://psalm.dev/docs/annotating_code/type_syntax/array_types/#phpdoc-syntax

Reply:
same for phpstan

Reply:
Sent PR: #8114